Android build failed with an exception

Hi dears.

I got an issue during Android build:

```log
...
Launching lib/main.dart on Pixel 7 in debug mode...

FAILURE: Build failed with an exception.

* Where:
Build file '~/.pub-cache/hosted/pub.dev/wakelock_plus-1.1.0/android/build.gradle' line: 30

* What went wrong:
A problem occurred evaluating project ':wakelock_plus'.
> Could not find method namespace() for arguments [dev.fluttercommunity.plus.wakelock] on extension 'android' of type com.android.build.gradle.LibraryExtension.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
BUILD FAILED in 1s
Exception: Gradle task assembleDebug failed with exit code 1
Exited (sigterm)
```

Could you help me please?

--- a/wakelock_plus/android/build.gradle
+++ b/wakelock_plus/android/build.gradle
@@ -27,7 +27,10 @@ apply plugin: 'kotlin-android'
 android {
     compileSdkVersion 33
 
-    namespace 'dev.fluttercommunity.plus.wakelock'
+    // Conditional for compatibility with AGP <4.2.
+    if (project.android.hasProperty("namespace")) {
+        namespace 'dev.fluttercommunity.plus.wakelock'
+    }
 
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8

--- a/wakelock_plus/example/android/app/build.gradle
+++ b/wakelock_plus/example/android/app/build.gradle
@@ -26,7 +26,12 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    namespace "dev.fluttercommunity.plus.wakelock_example"
+
+    // Conditional for compatibility with AGP <4.2.
+    if (project.android.hasProperty("namespace")) {
+        namespace "dev.fluttercommunity.plus.wakelock_example"
+    }
+
     compileSdkVersion 33
     ndkVersion flutter.ndkVersion
 

